### ISSUE TITLE
iOS fix defaulted to inadequate codec.

### ISSUE THREAD

**Issue by ozansarier** [ozansarier/enkazdinlemeuygulamasi#1]
The latest fix for ios at line:

406                  const blob = new Blob(chunks, { 'type': mediaRecorder.mimeType });

is inferior to the ogg recording of the previous release as it now gates very low level audio earlier.

Investigate a way to implement wav recording while maintaining iOS/Safari compatibility

const blob = new Blob(chunks, { 'type': 'audio/wav; codecs=MS-PCM' });

**Comment by ozansarier** [comment]
Another possible solution to gating :

Explicitly declare echoCancellation and noiseSupression false.


```
stream = await navigator.mediaDevices.getDisplayMedia(
            { 
                video: false, 
                audio: 
                    { 
                        channels: 2, 
                        autoGainControl: false, 
                        echoCancellation: false, 
                        noiseSuppression: false 
                    }
                }
            );
```

**Comment by ozansarier** [comment]
Apparently, gating is not an issue of codec and it is device specific. Still, recording uncompressed wav files will be beneficial. 

Disabling gating should also be inspected.

**Comment by ozansarier** [comment]
Fixed for now in the v0.5 update.

Disabling  autoGainControl   echoCancellation and noiseSuppression solved problems with android devices but created problems with Apple ones.

Added a script to detect os, then set the properties accordingly.
